vespajlib/src/test/java/com/yahoo/tensor/MixedTensorTestCase.java

```diff
@@ -159,4 +164,25 @@ public void testTwoMappedTwoIndexedBuilding() {
                 tensor.toString());
     }
 
+    @Test
+    public void testSplitIntoDense() {
```

Comment:
Unrelated I assume.
Looks like documentation for the code you gave to @andreer  the other day.

Reply:
Yes, decided it was worth keeping + lazy ...